supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/PipelinesApiInitializationActor.scala

```diff
@@ -125,6 +125,7 @@ class PipelinesApiInitializationActor(pipelinesParams: PipelinesApiInitializatio
     for {
       paths <- workflowPaths
       _ = publishWorkflowRoot(paths.workflowRoot.pathAsString)
+      _ <- Future.fromTry(GoogleLabels.fromWorkflowOptions(workflowOptions))
```

Comment:
I am not too sure what this line does... `fromWorkflowOptions` does not seem to side-effect anything and then we throw away its result.

If we are checking that it completes successfully but don't care about the result perhaps a comment would be appropriate so people don't think it's useless and delete it

Reply:
Agreed. Added a comment.